### PR TITLE
app-i18n/translate-shell: bump version, skip online tests

### PR DESCRIPTION
**Pull request by ferki**
It also contains the following changes:
 - cleanup metadata
 - add back proxy maintainers
 - switch to EAPI6

### CODE REVIEW

**Comment by ferki** [comment]
@gentoo/proxy-maint: could you take a look at this, please? I'm happy to update/squash commits if needed :)

**Comment by monsieurp** [comment]
Hi @ferki. Thanks for the PR. This package is also co-maintained by Patrick. He doesn't like Github very much and will most likely complain if we merge it. Could you please first get him to look at your patches and get him to approve them? Thanks!

**Comment by mgorny** [comment]
@ferki, sorry for the terrible delay. Could you file a bug for this, update the commit message to mention the bug per GLEP 66 and rebase the PR? Alternatively, just submit the changes as patches to the bug.

**Comment by ferki** [comment]
@mgorny: thanks for taking a look! There are already a couple of bugs in bugzilla that are fixed by these commits:
 - [Bug 617230](https://bugs.gentoo.org/show_bug.cgi?id=617230)
 - [Bug 630950](https://bugs.gentoo.org/show_bug.cgi?id=630950)

The commit messages are also already including those bugs with respective mentions/closes as per GLEP 66.

I have rebased now on top of current master, where the the only change was about the hash methods used in the Manifest file.

I took the chance to also bump the package once again to its latest version from my overlay, which meant a simple rename of the ebuild, and bumping the copyright year in the header (plus I squashed away the intermediate, unmerged bump commit).

I have also attached the latest version of the ebuild to the aforementioned bug about the version bump in bugzilla.

I'm looking forward to finally merge these, and let me know if you need any further changes!

File: app-i18n/translate-shell/Manifest [hunk 1]
```diff
@@ -1 +1 @@
-DIST translate-shell-0.9.6.2.tar.gz 69544 BLAKE2B b49a2c6d7101f5063c8c548c2dcd26ed8b5a310d5e723293b8082bb1bdffbb926fb6f0050953825c9fc4567880cfbf1e9956236db3c28b176db851983c743d0d SHA512 a7bd3fba43485a22c2fdd4261c24579b03aeafefa40863c3b8eb1c814a167b9a6f5700086c1cd411063c66c169875b96d2634ed04f45beb6d53b4725eb7fbea9
+DIST translate-shell-0.9.6.4.tar.gz 69788 SHA256 5ac8fd1a6b772750942f92665966ddf2b825594eb969770aabf16d7a83b3e6b9 SHA512 98c4a329794d38a5d22d98bf7bb821f1adc3e42dac96d2fd791e7cf64d785458d0f6006e83b21497f259525b1a70f306ab69090d3a12996610f742323a371147 WHIRLPOOL 53a9561738356d3ff101bd11ef96792d5cba5419a6db525a467fc3ff924ce97174f337ff8b7359ed3b58466280cf712d043973d48310c21c91e9914f934d7f08
```
Comment (mgorny) [inline app-i18n/translate-shell/Manifest]: 1. I think you want `Closes:`, not `Bug:` here.
2. Please split removal from bump into separate commits.
3. This is the wrong hash set, so please regenerate the Manifest. You will need to remove the old entry to make Portage do the right thing.

File: app-i18n/translate-shell/Manifest [hunk 2]
```diff
@@ -1 +1 @@
-DIST translate-shell-0.9.6.4.tar.gz 69788 SHA256 5ac8fd1a6b772750942f92665966ddf2b825594eb969770aabf16d7a83b3e6b9 SHA512 98c4a329794d38a5d22d98bf7bb821f1adc3e42dac96d2fd791e7cf64d785458d0f6006e83b21497f259525b1a70f306ab69090d3a12996610f742323a371147 WHIRLPOOL 53a9561738356d3ff101bd11ef96792d5cba5419a6db525a467fc3ff924ce97174f337ff8b7359ed3b58466280cf712d043973d48310c21c91e9914f934d7f08
+DIST translate-shell-0.9.6.6.tar.gz 73189 BLAKE2B ad4de403f2c95a1e4de5c963cb017f5a744f08d3bb8256dcc22f28c0916f6a0e693bce171a5dc500de5353c954bab7681f1227df9e48be286ad5aa07c5d06fa4 SHA512 c5451e537f44e325ee2f59d54942f91b8e2ccc0e322c42a2c4a0e98b8a933fe0630798a45e6f587ad5aee2cbb092204710ae36779c94b8333376a8072678d868
```
Comment (mgorny) [inline app-i18n/translate-shell/Manifest]: Ok, please squash this with the earlier commit, i.e. do not introduce-then-remove intermediate versions.

**Comment by ferki** [comment]
@mgorny: so I have rewritten the git history based on your feedback, so now there's only one version bump, and then one removal of old ebuild later, while I tried to preserve as much of the history as I could.

This also fixed the Manifest entries, as well as avoiding multiple references to the same bugzilla issues (once with `Closes:` and once with `Bug:`). Copyright headers should also be OK, since old files are not being edited, and new ones already had the right header.

I really appreciate someone's finally providing feedback on this one, so just let me know if you need me to do more changes or history cleanup.

**Comment by mgorny** [comment]
Thanks. The only remaining issue I see is that it would be nice if the version bump commit had the version in commit message summary but I can fix that while merging. I'll test and merge later today.

**Comment by ferki** [comment]
I have added the version to the bump commit message. Feel free to rewrite it upon merge if you still need to change it.

**Comment by mgorny** [comment]
Boo, it looks that either that 'skip' does not work or you didn't catch all the tests:

```
thing()
[PASS] ( 3/3 ) TESTING anything()
[PASS] ( 1/4 ) TESTING exists()
[PASS] ( 2/4 ) TESTING exists()
[PASS] ( 3/4 ) TESTING exists()
[PASS] ( 4/4 ) TESTING exists()
[PASS] ( 1/3 ) TESTING belongsTo()
[PASS] ( 2/3 ) TESTING belongsTo()
[PASS] ( 3/3 ) TESTING belongsTo()
[PASS] ( 1/4 ) TESTING identical()
[PASS] ( 2/4 ) TESTING identical()
[PASS] ( 3/4 ) TESTING identical()
[PASS] ( 4/4 ) TESTING identical()
[PASS] ( 1/1 ) TESTING append()
[PASS] ( 1/4 ) TESTING isnum()
[PASS] ( 2/4 ) TESTING isnum()
[PASS] ( 3/4 ) TESTING isnum()
[PASS] ( 4/4 ) TESTING isnum()
[PASS] ( 1/3 ) TESTING startsWithAny()
[PASS] ( 2/3 ) TESTING startsWithAny()
[PASS] ( 3/3 ) TESTING startsWithAny()
[PASS] ( 1/4 ) TESTING matchesAny()
[PASS] ( 2/4 ) TESTING matchesAny()
[PASS] ( 3/4 ) TESTING matchesAny()
[PASS] ( 4/4 ) TESTING matchesAny()
[PASS] ( 1/4 ) TESTING replicate()
[PASS] ( 2/4 ) TESTING replicate()
[PASS] ( 3/4 ) TESTING replicate()
[PASS] ( 4/4 ) TESTING replicate()
[PASS] ( 1/4 ) TESTING reverse()
[PASS] ( 2/4 ) TESTING reverse()
[PASS] ( 3/4 ) TESTING reverse()
[PASS] ( 4/4 ) TESTING reverse()
[PASS] ( 1/4 ) TESTING join()
[PASS] ( 2/4 ) TESTING join()
[PASS] ( 3/4 ) TESTING join()
[PASS] ( 4/4 ) TESTING join()
[PASS] ( 1/1 ) TESTING explode()
[PASS] ( 1/8 ) TESTING escapeChar()
[PASS] ( 2/8 ) TESTING escapeChar()
[PASS] ( 3/8 ) TESTING escapeChar()
[PASS] ( 4/8 ) TESTING escapeChar()
[PASS] ( 5/8 ) TESTING escapeChar()
[PASS] ( 6/8 ) TESTING escapeChar()
[PASS] ( 7/8 ) TESTING escapeChar()
[PASS] ( 8/8 ) TESTING escapeChar()
[PASS] ( 1/6 ) TESTING literal()
[PASS] ( 2/6 ) TESTING literal()
[PASS] ( 3/6 ) TESTING literal()
[PASS] ( 4/6 ) TESTING literal()
[PASS] ( 5/6 ) TESTING literal()
[PASS] ( 6/6 ) TESTING literal()
[PASS] ( 1/4 ) TESTING escape()
[PASS] ( 2/4 ) TESTING escape()
[PASS] ( 3/4 ) TESTING escape()
[PASS] ( 4/4 ) TESTING escape()
[PASS] ( 1/4 ) TESTING unescape()
[PASS] ( 2/4 ) TESTING unescape()
[PASS] ( 3/4 ) TESTING unescape()
[PASS] ( 4/4 ) TESTING unescape()
[PASS] ( 1/10) TESTING parameterize()
[PASS] ( 2/10) TESTING parameterize()
[PASS] ( 3/10) TESTING parameterize()
[PASS] ( 4/10) TESTING parameterize()
[PASS] ( 5/10) TESTING parameterize()
[PASS] ( 6/10) TESTING parameterize()
[PASS] ( 7/10) TESTING parameterize()
[PASS] ( 8/10) TESTING parameterize()
[PASS] ( 9/10) TESTING parameterize()
[PASS] (10/10) TESTING parameterize()
[PASS] ( 1/10) TESTING unparameterize()
[PASS] ( 2/10) TESTING unparameterize()
[PASS] ( 3/10) TESTING unparameterize()
[PASS] ( 4/10) TESTING unparameterize()
[PASS] ( 5/10) TESTING unparameterize()
[PASS] ( 6/10) TESTING unparameterize()
[PASS] ( 7/10) TESTING unparameterize()
[PASS] ( 8/10) TESTING unparameterize()
[PASS] ( 9/10) TESTING unparameterize()
[PASS] (10/10) TESTING unparameterize()
[PASS] ( 1/4 ) TESTING toString()
[PASS] ( 2/4 ) TESTING toString()
[PASS] ( 3/4 ) TESTING toString()
[PASS] ( 4/4 ) TESTING toString()
[PASS] ( 1/4 ) TESTING squeeze()
[PASS] ( 2/4 ) TESTING squeeze()
[PASS] ( 3/4 ) TESTING squeeze()
[PASS] ( 4/4 ) TESTING squeeze()
[PASS] ( 1/12) TESTING yn()
[PASS] ( 2/12) TESTING yn()
[PASS] ( 3/12) TESTING yn()
[PASS] ( 4/12) TESTING yn()
[PASS] ( 5/12) TESTING yn()
[PASS] ( 6/12) TESTING yn()
[PASS] ( 7/12) TESTING yn()
[PASS] ( 8/12) TESTING yn()
[PASS] ( 9/12) TESTING yn()
[PASS] (10/12) TESTING yn()
[PASS] (11/12) TESTING yn()
[PASS] (12/12) TESTING yn()
[PASS] ( 1/4 ) TESTING quote()
[PASS] ( 2/4 ) TESTING quote()
[PASS] ( 3/4 ) TESTING quote()
[PASS] ( 4/4 ) TESTING quote()
[PASS] ( 1/6 ) TESTING unquote()
[PASS] ( 2/6 ) TESTING unquote()
[PASS] ( 3/6 ) TESTING unquote()
[PASS] ( 4/6 ) TESTING unquote()
[PASS] ( 5/6 ) TESTING unquote()
[PASS] ( 6/6 ) TESTING unquote()
[PASS] ( 1/4 ) TESTING fileExists()
[PASS] ( 2/4 ) TESTING fileExists()
[PASS] ( 3/4 ) TESTING fileExists()
[PASS] ( 4/4 ) TESTING fileExists()
[PASS] ( 1/4 ) TESTING dirExists()
[PASS] ( 2/4 ) TESTING dirExists()
[PASS] ( 3/4 ) TESTING dirExists()
[PASS] ( 4/4 ) TESTING dirExists()
[PASS] ( 1/2 ) TESTING detectProgram()
[PASS] ( 2/2 ) TESTING detectProgram()
[PASS] (121/121) TESTING COMPLETED. PERFECT!
====== TESTING FOR Utils.awk STARTED
[PASS] ( 1/1 ) TESTING GawkVersion
[PASS] ( 1/1 ) TESTING Rlwrap
[PASS] ( 1/1 ) TESTING Emacs
[PASS] ( 1/5 ) TESTING newerVersion()
[PASS] ( 2/5 ) TESTING newerVersion()
[PASS] ( 3/5 ) TESTING newerVersion()
[PASS] ( 4/5 ) TESTING newerVersion()
[PASS] ( 5/5 ) TESTING newerVersion()
[FAIL] ( 1/1 ) TESTING curl()
       assertEqual:  IS NOT EQUAL TO https://httpbin.org/get
[FAIL] ( 1/1 ) TESTING curlPost()
       assertEqual:  IS NOT EQUAL TO https://httpbin.org/post
[PASS] ( 1/3 ) TESTING dump()
[PASS] ( 2/3 ) TESTING dump()
[PASS] ( 3/3 ) TESTING dump()
[PASS] ( 1/1 ) TESTING base64()
[PASS] ( 1/1 ) TESTING uprintf
[FAIL] (13/15) TESTING COMPLETED
       13 PASSED, 2 FAILED
make: *** [Makefile:26: test] Error 1

```

**Comment by ferki** [comment]
Well, it seems they have added more online tests since I created the original patch back in September. I'll push changes to skip the new ones too.

**Comment by mgorny** [comment]
Thanks. Please leave a comment once you do that since GH may not inform me of new commits.

**Comment by ferki** [comment]
@mgorny: it seems more tests needs to be skipped in more files, so I decided to use apply a proper patch instead of monkey-patching the `Makefile` with `sed`.

Supporting offline tests feels like a good candidate to send upstream, but I'd prefer not to hold back this PR any further if it's in an acceptable state otherwise.

File: app-i18n/translate-shell/translate-shell-0.9.6.6.ebuild [hunk 3]
```diff
@@ -28,6 +28,8 @@ DEPEND="${RDEPEND}
 	test? ( app-editors/emacs )
 	"
 
+PATCHES="${FILESDIR}/${P}-remove-online-tests.patch"
```
Comment (mgorny) [inline app-i18n/translate-shell/translate-shell-0.9.6.6.ebuild]: This should be an array. I'll fix it.

**Comment by ferki** [comment]
Thanks for both the feedback and the fix, as well as for the merge!
